src/content/tools/doc-comment-references.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Documentation comment references
+short-title: Comment references
+description: Learn about doc comment references and their syntax.
+---
+
+Doc comments can contain references to various code elements. A library member
+or class member, etc. can be referenced by wrapping its name in square brackets
+(`[...]`) in a doc comment (a comment starting with `///`). Examples:
+
+```dart
+/// Returns a [String].
+String f() => 'Hello';
+
+/// Wraps [object] with [Future.value].
+Future<T> g<T>(T object) => Future.value(object);
+```
+
+These doc comments contain references to the `String` class, the `object`
+parameter, and the `Future.value` constructor.
+
+## Features of references
+
+There are several benefits to referring to a code element in a doc comment as a
+doc comment reference:
+
+### Editor features
```

Comment:
Perhaps the end of this section or a tip aside can call out the [`comment_references`](https://dart.dev/lints/comment_references) lint as enabling verification of them?

```
:::tip
<Tip content goes here>
:::
```

Another option would be calling this out in the "What can be referenced" section.

Reply:
Done, love it.